private/redact-protected-keys.js

```diff
@@ -44,6 +44,9 @@ module.exports = function redactProtectedKeys(dictionary, blacklist) {
     throw new Error('Consistency violation: Unexpected bad usage in cleanseDictionary.  Expected blacklist to be an array of strings, but got: '+util.inspect(blacklist,{depth:null}));
   }//>-•
 
+  if(_.isUndefined(recursive)){
```

Comment:
@dan-omniscience yeah, if you wouldn't mind, that'd be helpful just for consistency 
@tarlepp thanks, good catch